test/failing.verification

```diff
@@ -0,0 +1,93 @@
+# Not sure why they're not passing, looks like a whitespace issue in the diff?
+test/operational/control-flow/while-no-invar.bpl
+test/operational/control-flow/square-proc.bpl
```

Comment:
@nishantjr These test _should_ pass but don't. It looks like the only diff by `diff-kboogie` is whitespace. I'm not sure what's going on there could you have a look?

Reply:
Looks like a bug in substitution actually.

Reply:
You mean the substitution being done by `diff-kboogie`? or one of the substitution rules in the `k` definition?

Reply:
I thought it was the one we defined in the boogie repository. But turns out, I wasn't evaluating the arguments of the function to values before applying them. Pushed a fix

Reply:
The "fix" breaks things more, since functions with symbolic aren't evaluated. :shrug:

Reply:
All right. Well, we can just revert the "fix" so that we can merge this with `master` with all tests passing, then worry about it later.